### PR TITLE
Added config option and flag 'showdiff'.

### LINKED ISSUE #43: Request: Show diff icon
It would be nice if there was a diff icon (Diff icon) in the pagelist that would take you to the diff page.

### PR DESCRIPTION
**Pull request by LarsGit223**
This enables displaying of the column 'Differences' which shows the diff icon with a link to the diff page of the page in the row. Closes #43.

### CODE REVIEW

File: helper.php [hunk 1]
```diff
@@ -392,6 +397,23 @@ function _descCell() {
...
+    function _diffCell($id) {
+        // check for page existence
+        if (!isset($this->page['exists'])) {
+            if (!isset($this->page['file'])) $this->page['file'] = wikiFN($id);
+            $this->page['exists'] = @file_exists($this->page['file']);
+        }
+
+        // produce output
+        $content = '<a href="'.wl($id).($this->page['section'] ? '#'.$this->page['section'] : '').'&do=diff" class="diff_link">
+<img src="/lib/images/diff.png" width="15" height="11" title="Show differences to current revisions" alt="Show differences to current revisions"/>
```
Comment (michitux) [inline helper.php]: Please add these strings to the language files such that they can be translated.

File: helper.php [hunk 2]
```diff
@@ -392,6 +397,23 @@ function _descCell() {
...
+     */
+    function _diffCell($id) {
+        // check for page existence
+        if (!isset($this->page['exists'])) {
+            if (!isset($this->page['file'])) $this->page['file'] = wikiFN($id);
+            $this->page['exists'] = @file_exists($this->page['file']);
+        }
+
+        // produce output
+        $content = '<a href="'.wl($id).($this->page['section'] ? '#'.$this->page['section'] : '').'&do=diff" class="diff_link">
```
Comment (michitux) [inline helper.php]: The anchor needs to be after the get parameters. In order to ensure correct parameter separators please pass the diff parameter to the `wl`-function.

File: helper.php (hunk 1 above)
Comment (LarsGit223) [inline helper.php]: Done.

File: helper.php (hunk 2 above)
Comment (LarsGit223) [inline helper.php]: Done.

**Comment by michitux** [comment]
Could you please increase the version and update the wiki page with the new version and some documentation of the new feature? Thank you!

**Comment by LarsGit223** [comment]
Done.

**Pull request merged**